external/llvm-project/mlir/test/Dialect/Tosa/tosa-partition.mlir

```diff
@@ -88,10 +88,10 @@ func.func @test_fusion6(%arg0: tensor<128x8x32x32xf32>, %arg1: tensor<128x8x3x3x
 }
 
 // CHECK-LABEL: func private @test_fusion7__part_0
+// CHECK-NEXT: tosa.abs
```

Comment:
Why'd the abs() move before the conv2d here? Also, we don't currently have load-side fusion in RocMLIR ... but maybe we won't hit this case by setting trailing-only in our pipelines.

Reply:
I'm running it with trailing-only=false, which was the default until recently.  That gathers the abs into the leadingOps for the conv2d and thus includes it in the __part function.

Simon's recent patch changed the default because it fit his larger cases better;  I still wanted to test the leading-ops functionality in these little tests.